android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -132,6 +133,8 @@
                    BraveRewardsObserver, BraveRewardsNativeWorker.PublisherObserver {
     public static final String PREF_HIDE_BRAVE_REWARDS_ICON = "hide_brave_rewards_icon";
     private static final String JAPAN_COUNTRY_CODE = "JP";
+    private static final List<String> mBraveSearchEngineDefaultRegions =
+            Arrays.asList("CA", "DE", "FR", "GB", "US");
```

Comment:
Besides these, I think we also need remove the `US`/`CA` restriction we added in https://github.com/brave/brave-core/pull/8966/commits/4808157d3bab53e5c0509808f20c295ee124e247 (to `android/java/org/chromium/chrome/browser/onboarding/SearchEngineOnboardingFragment.java`)

That way we can show Brave Search for all other regions (which don't have Brave Search as default)

Reply:
Actually - followed up with @ecnmst - no change needed here for now 😄